### PR TITLE
Support translating @tparam to @param

### PR DESCRIPTION
**Pull request by manuzhang**

### CODE REVIEW

File: plugin/src/main/scala/com/typesafe/genjavadoc/BaseComments.scala [hunk 1]
```diff
@@ -19,7 +19,8 @@ trait BaseComments { this: TransformCake ⇒
     "<p/>" -> "<p></p>",
     "=>" -> "=&gt;",
     "[[" -> "{@link ",
-    "]]" -> "}")
+    "]]" -> "}",
+    "@tparam" -> "@param")
```
Comment (patriknw) [inline plugin/src/main/scala/com/typesafe/genjavadoc/BaseComments.scala]: ah, nice

**Comment by ktoso** [comment]
Wait a second, that's not quite correct yet.
In ScalaDoc what is `@tparam T` is in JavaDoc `@param <T>`, see scaladoc reference docs: http://docs.oracle.com/javase/7/docs/technotes/tools/windows/javadoc.html#param

The current conversion would likely cause javadoc validation errors, since "parameter not found" etc. Could you add a test and look into it a bit more @manuzhang ?

**Comment by manuzhang** [comment]
@ktoso No, it's not right but there are no validation errors. By the way, I find that parameter tag `@param T` is not translated to `@param <T>` either.

**Comment by ktoso** [comment]
`param` is `param`, so why would it be translated to `<T>`, it's just tparam that needs this modification

**Comment by manuzhang** [comment]
Ah, my mistake. Let me fix it.

**Comment by manuzhang** [comment]
@patriknw @ktoso Another try. Please review

**Comment by manuzhang** [comment]
Any comments for the latest version ?

File: plugin/src/test/resources/expected_output/basic/akka/rk/buh/is/it/Y.java [hunk 2]
```diff
@@ -1,3 +1,6 @@
 package akka.rk.buh.is.it;
+/**
+ * @param <A>
```
Comment (ktoso) [inline plugin/src/test/resources/expected_output/basic/akka/rk/buh/is/it/Y.java]: exactly, cool :)

Comment (ktoso) [reply plugin/src/test/resources/expected_output/basic/akka/rk/buh/is/it/Y.java]: Let's add a description though to know if it actually completely works

**Comment by ktoso** [comment]
I've added a type parameter description so that the test is "complete", see: https://github.com/typesafehub/genjavadoc/pull/110/commits/e6b03b83a5b7564411ec0a45f61879c175f71505

Waiting on validation

**Comment by manuzhang** [comment]
@ktoso Thanks, I missed that part and fixed it with f31fe1b

**Comment by ktoso** [comment]
Excellent, now it's all good 👍  Thanks a ton!

**Pull request merged**